nxrm-ha-helm/values.yaml

```diff
@@ -0,0 +1,208 @@
+# Declare variables to be passed into your templates.
+namespaces:
+  nexusNs: nexusrepo
+  cloudwatchNs: amazon-cloudwatch
+  externaldnsNs: nexus-externaldns
+azure:
+  enabled: false  #set to true to enable aws specific yamls/snippets
+  keyvault:
+    enabled: false # set to true to pull license, db and nexus admin password secrets from azure key vault
+aws:
+  enabled: false #set to true to enable aws specific yamls like external dns and fluent bit
+  secretmanager:
+    enabled: false # set to true to pull license, db and nexus admin password secrets from aws secret manager
+  externaldns:
+    enabled: false
+    domainFilter: example.com #your root domain e.g example.com
+    awsZoneType: private # hosted zone to look at (valid values are public, private or no value for both)
+    clusterRegion: us-east-1
```

Comment:
Should we define the region in one location?

Reply:
https://github.com/sonatype/nxrm3-ha-repository/pull/21/commits/114194c5ea723ea8722a9009ddb2d502fd06cb79